src/sections/Hero/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import LangSwitcher from '../../components/LangSwitcher/LangSwitcher'
+import logo from '../../assets/logo.svg'
+import gifAdult from '../../assets/bg-animation.gif'
+import gifCh from '../../assets/bg-animation-ch.gif'
+import flowers from '../../assets/flowers.svg'
+import flowersMobile from '../../assets/flowers-2.svg'
+import './styles.scss'
+
+const Hero: FC = () => {
+	const { t, i18n } = useTranslation()
+
+	const getCurrentGif = () => {
+		if (i18n.language === 'ad') {
+			return gifAdult
+		} else return gifCh
+	}
+
+	return (
+		<section className="hero">
+			{i18n.language === 'ch' && (
+				<img
+					alt=""
+					src={window.innerWidth <= 767 ? flowersMobile : flowers}
+					className={window.innerWidth <= 767 ? 'hero__bg-flowers-mobile' : 'hero__bg-flowers'}
+				/>
+			)}
+			<img
+				src={getCurrentGif()}
```

Comment:
it may be better to just use the link as a variable in the locales folder and use only 1 img tag

Reply:
I'm not sure that it's going to work after building the project but I may try it anyway if it's required.

Reply:
Btw, there is only one img tag now